### PR TITLE
media-queries-for-desktop-and-responsivenes

### PR DESCRIPTION
**Pull request by piesieomane**
This pull request continues with the project of adding media queries to the desktop version. Some changes were made to the mobile version to bring about the responsiveness of the whole project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -237,6 +255,7 @@ <h4>Skills</h4>
                 </div>
             </li>
         </ul>
+        </div>
     </section>
     <!-- contact-form -->
     <section class="contact-me">
```
Comment (danmainah) [inline index.html:261]: - [ ] Kindly check on your footer design some background elements are missing, below is a comparison of your design with the Figma design;

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image|  image

File: index.html [hunk 2]
```diff
@@ -20,6 +20,13 @@ <h3 class="my-logo">My logo</h3>
                <img src="images/Enabled.png" alt="normal button">
            </div>
         </nav>
+        <nav class="nav-links nav-links1">
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
```
Comment (danmainah) [inline index.html:29]: - [ ] Kindly check your font color for your nav items, it differs from the design given, also some background images are missing, check the comparison below for better understanding;

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @piesieomane , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] Kindly update your readme file getting started section, it should reflect on your current project not hello world project.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AshakaE — APPROVED** [review]
Hi @piesieomane , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**